example.js

```diff
@@ -0,0 +1,29 @@
+const irrigation = require('./irrigation')
```

Comment:
Oops. I didn't mean to add this file. This code exists in the README. I just put it here to make sure it was formatted properly. I'll delete before merging.